ui/src/router/components/form/components/router_config/rule_card/RuleCard.js

```diff
@@ -69,6 +71,12 @@ export const RuleCard = ({
 
   // Hide last row when all routes are selected
   const routeSelectionOptions = (rule.routes.length < routes.length ? [...rule.routes,  "_none_"]: rule.routes);
+  
+  const conditionLabel = protocol === "UPI_V1" ? 
+  <FormLabelWithToolTip
+    label="Conditions *"
+    content='Prediction Context of the UPI proto.  e.g. "model_name" in "model_a"'
```

Comment:
Why is this here? The condition can also be for the header, right?

<img width="752" alt="Screenshot 2022-10-27 at 9 29 10 AM" src="https://user-images.githubusercontent.com/23465343/198169813-ee123e78-27ad-489a-a04f-bfeea90920d8.png">

IMO it's better for such descriptions to be a tooltip on the option itself, rather than in the parent UI component - it keeps the parent component generic.

Reply:
ah this is a mistake, initially i forgotten that we support `Header` as well and so I written with assumption `Prediction Context` was the only option. 

`tooltip on the option itself` -> right, this sounds better. will change it

Reply:
Not sure if it helps but i shifted it 😅 
<img width="532" alt="image" src="https://user-images.githubusercontent.com/30390872/198189102-bf2af7f4-d08e-47f7-9d7c-01ee6c883794.png">